week2/homework/src/index.js

```diff
@@ -1,3 +1,53 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+const program = require('commander');
+const commands = require('./commands');
+const addToDo = require('./addToDo');
+const showList = require('./showList');
+const remove = require('./removeToDo');
+const reset = require('./resetList');
+if (process.argv.length === 2) {
+  commands.showCommandsList();
+}
+ else {
```

Comment:
this is somehow a long `else` close, perhaps we could simplify it by looking to it's actions:
 - we want a help on `help` command
 - we want a help on an empty submit

from the first we could:
```javascript
program.addImplicitHelpCommand = () => {};
```
And the second: 
```javascript
if (process.argv.length === 2) {
  process.argv.push('help')
}
```

references: [first], the [second].

[first]: https://github.com/tj/commander.js/issues/683#issuecomment-339602122
[second]: https://github.com/tj/commander.js/issues/303#issuecomment-477441964